{
  "id": "check_expected_failure",
  "tests": [
    {
      "code": "raise ValueError",
      "type": "python",
      "shouldfail": true
    },
    {
      "command": "funkyfailure_somecommand",
      "type": "shell",
      "shouldfail": true
    }
  ],
  "version": 0
}
